api/models/vpp/interfaces/interface.proto

```diff
@@ -226,3 +227,16 @@ message BondLink {
     }
     repeated BondedInterface bonded_interfaces = 12;
 }
+
+message GreLink {
+    enum Type {
+        L3 = 0; // L3 GRE (i.e. this tunnel is in L3 mode)
+        TEB = 1; // Transparent Ethernet Bridging - the tunnel is in L2 mode
+        ERSPAN = 2; // ERSPAN - the tunnel is for port mirror SPAN output
+    };
+    Type tunnel_type = 1;
+    string src_addr = 2; 
+    string dst_addr = 3;
+    uint32 outerFibID = 4;
```

Comment:
[Fields should use `snake_case`.](https://developers.google.com/protocol-buffers/docs/style#message-and-field-names)